v2/fleet-local/control/zk-exhibitor@.service

```diff
@@ -25,6 +25,7 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
   -e S3_BUCKET=`etcdctl get /zookeeper/s3_exhibitor_bucket` \
   -e S3_PREFIX=`etcdctl get /zookeeper/s3_exhibitor_prefix` \
   -e ZK_ENSEMBLE_SIZE=`etcdctl get /zookeeper/ensemble_size` \
+  -e ZK_PASSWORD=`etcdctl get /zookeeper/ZK_PASSWORD` \
```

Comment:
And lastly I did `core@ip-10-43-4-125 sandbox-anshu-stack-us-east-1e-control ~ $ etcdctl rm zookeeper/ZK_PASSWORD
PrevNode.Value: 
core@ip-10-43-4-125 sandbox-anshu-stack-us-east-1e-control ~ $ etcdctl get zookeeper/ZK_PASSWORD
Error:  100: Key not found (/zookeeper/ZK_PASSWORD) [208101]`

I was able to access the console without being prompted for username and password